### PR TITLE
Improve error messages for sorting

### PR DESCRIPTION
**Pull request by lawwm**
Improve the error messages for sorting.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/SortCommand.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
             + "Example: " + COMMAND_WORD + " " + PREFIX_TAG + " " + PREFIX_NAME
             + " " + PREFIX_ORDER + "desc" + "\n";
 
-    public static final String MESSAGE_SUCCESS = "Sorted Modules successfully: %s";
+    public static final String MESSAGE_SUCCESS = "Sorted successfully by: %s";
```
Comment (fantablack) [inline src/main/java/seedu/address/logic/commands/SortCommand.java:37]: Acknowledged fix for inaccurate display of "Sorted by Modules..." from #51

File: src/main/java/seedu/address/logic/parser/SortCommandParser.java [hunk 2]
```diff
@@ -36,7 +36,7 @@ public SortCommand parse(String args) throws ParseException {
                 PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_STATUS, PREFIX_TAG);
 
         if (prefixes.size() == 0) {
-            throw new ParseException(MESSAGE_NO_PARAMETERS_SUPPLIED);
+            throw new ParseException(String.format(MESSAGE_NO_PARAMETERS_SUPPLIED, SortCommand.MESSAGE_USAGE));
```
Comment (fantablack) [inline src/main/java/seedu/address/logic/parser/SortCommandParser.java:39]: Acknowledged enhancement for displaying example usages in addition to error message as outlined in #51

**Review by fantablack — COMMENTED** [review]
Tested and verified these fixes addressing both concerns in #51

Adequate but a small improvement would be showing the successful sorted flag i.e. Sorted successfully by: NAME instead of just [n/].
I might implement this later.

LGTM

**Comment by bakano98** [comment]
LGTM! Thanks for implementing this 😄

**Comment by lawwm** [comment]
Implement displaying success fields.
image

**Review by Btaykb — COMMENTED** [review]
The success message is neat and clear, LGTM

**Pull request merged**